glava/render.c

```diff
@@ -2073,22 +2073,22 @@ bool rd_update(struct renderer* r, float* lb, float* rb, size_t bsz, bool modifi
 
     /* Handle counters and print FPS counter (if needed) */
 
-    ++gl->fcounter;           /* increment frame counter                          */
-    if (modified) {           /* if this is an update/key frame                   */
-        ++gl->ucounter;       /*   increment update frame counter                 */
-        gl->kcounter = 0;     /*   reset keyframe counter (for interpolation)     */
-    } else ++gl->kcounter;    /* increment keyframe counter otherwise             */
-    gl->tcounter += duration; /* timer counter, measuring when a >1s has occurred */
-    if (gl->tcounter >= 1.0D) {
-        int elapsed_frames = gl->fcounter - gl->fcounter_last;
-        gl->fr = elapsed_frames / gl->tcounter; /* frame rate (FPS)     */
-        gl->ur = gl->ucounter / gl->tcounter;   /* update rate (UPS)    */
-        if (gl->print_fps)                      /* print FPS            */
-            printf("FPS: %.2f, UPS: %.2f, frame: %d\n",
-                   (double) gl->fr, (double) gl->ur, (int) gl->fcounter);
-        gl->tcounter = 0;                     /* reset timer                     */
-        gl->fcounter_last = gl->fcounter;     /* save this frame for next update */
-        gl->ucounter = 0;                     /* reset update counter            */
+    ++gl->fcounter;                   /* increment frame counter                          */
+    if (modified) {                   /* if this is an update/key frame                   */
+        ++gl->ucounter;               /*   increment update frame counter                 */
+        gl->kcounter = 0;             /*   reset keyframe counter (for interpolation)     */
+    } else ++gl->kcounter;            /* increment keyframe counter otherwise             */
+    gl->tcounter += (float) duration; /* timer counter, measuring when a >1s has occurred */
```

Comment:
I'm not sure if this double to float cast is okay, but I didn't find any problems with it while debugging.